client/constants/messages.js

```diff
@@ -45,12 +45,24 @@ const messages = {
   FETCH_SYNC_RECORDS: _, /* @param Array.<string> categoryNames, @param {number} startAt (in seconds) */
   /**
    * webview -> browser
-   * browser must update its local values with the new sync records, performing
-   * conflict-resolution as necessary.
+   * browser should find objects with matching objectIds then call
+   * RESOLVE_SYNC_RECORDS to perform conflict resolution.
    */
   RECEIVE_SYNC_RECORDS: _, /* @param {string} categoryName, @param {Array.<Object>} records */
```

Comment:
i see, that means `RESOLVE_SYNC_RECORDS` and `RESOLVED_SYNC_RECORDS` will only ever send records in the same category, right? so it seems those should both send a categoryName parameter